composer/trainer/trainer.py

```diff
@@ -1214,6 +1223,12 @@ def __init__(
                 raise ValueError(
                     'The `run_name` must be specified when using autoresume so Event.INIT is run with the correct run name.'
                 )
+            if any(
+                    isinstance(logger_destination, RemoteUploaderDownloader) and
+                    logger_destination._num_concurrent_uploads != 1 for logger_destination in self.logger.destinations):
+                raise ValueError(
+                    'Multiple concurrent uploads is not currently supported when using autoresume. Please set `num_concurrent_uploads` to 1 '
+                    'for all `RemoteUploaderDownloader` instances.')
```

Comment:
should we just... remove functionality to set more than 1 worker for now? like keep class functionality, but delete argument? is there ever a case where >1 worker is preferred given risk of race conditions?

Reply:
It _only_ breaks the symlink. So I think we should leave it in and figure out the real fix.